base/gfspvgmgr/virtual_group_manager.go

```diff
@@ -345,7 +350,11 @@ func (vgm *virtualGroupManager) refreshMetaByChain() {
 func (vgm *virtualGroupManager) PickVirtualGroupFamily() (*vgmgr.VirtualGroupFamilyMeta, error) {
 	vgm.mutex.RLock()
 	defer vgm.mutex.RUnlock()
-	return vgm.vgfManager.pickVirtualGroupFamily()
+	filter, err := vgm.genVgfFilter()
```

Comment:
Do we have to `genVgfFilter` every time `PickVirtualGroupFamily`? as `genVgfFilter` will start a slow RPC query, and the result is rarely changed.